terraform/modules/kubernetes/variables.tf

```diff
@@ -84,8 +110,8 @@ locals {
     var.app_environment_variables,
     {
       SERVICE_TYPE     = "web"
-      CUSTOM_HOSTNAME  = local.hostname
-      AUTHORISED_HOSTS = local.hostname
+      CUSTOM_HOSTNAME  = var.gov_uk_host_names == [] ? local.hostname : var.gov_uk_host_names[0]
```

Comment:
As discussed, CUSTOM_HOSTNAME and AUTHORISED_HOSTS can be removed so we get the values from keyvault explicitly

Reply:
removed